### PR TITLE
refactor: move bidi CDPSession

### PR DESCRIPTION
**Pull request by jrandolf-google**

### CODE REVIEW

**Comment by OrKoN** [comment]
Let's not land refactoring before the release is out

**Comment by OrKoN** [comment]
@Lightning00Blade PTAL

File: packages/puppeteer-core/src/bidi/CDPSession.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    }
+  }
+
+  override connection(): CdpConnection | undefined {
+    return undefined;
+  }
+
+  override async send<T extends keyof ProtocolMapping.Commands>(
+    method: T,
+    ...paramArgs: ProtocolMapping.Commands[T]['paramsType']
```
Comment (Lightning00Blade) [inline packages/puppeteer-core/src/bidi/CDPSession.ts]: Nit: Just to fix the correct type from the base class if its a no-op change.
```suggestion
    params?: ProtocolMapping.Commands[T]['paramsType'][0],
```

**Review by Lightning00Blade — APPROVED** [review]
LGTM

**Pull request merged**